kapitan/resources.py

```diff
@@ -99,6 +102,19 @@ def jinja2_render_file(search_paths, name, ctx):
 
     raise IOError("jinja2 failed to render, could not find file: {}".format(_full_path))
 
+def yaml_to_json(search_paths, name):
+    """return content of file in json"""
+    for path in search_paths:
+        _full_path = os.path.join(path, name)
+        logger.debug("yaml_to_json trying file %s", _full_path)
+        if os.path.exists(_full_path):
+            print(_full_path)
+            logger.debug("yaml_to_json found file at %s", _full_path)
+            try:
+                with open(_full_path) as f:
+                    return json.dumps(yaml.safe_load(f.read()))
```

Comment:
This will basically return the first result and immediately exit the for loop. So multiple search paths won't be meaningful here.

Reply:
I disagree, this seems correct as it needs to search through because that's how jsonnet search paths work. We do the same on other file based functions - you can't guarantee the file will be found in the first path.

Reply:
Got it, sounds good then. :)